docs_src/manuals/introduction/introduction.rst

```diff
@@ -50,160 +50,180 @@ computations in order to be optimal on the target hardware.
 
 .. code-block:: shell
 
  git clone http://github.com/eth-cscs/gridtools.git
  cd gridtools
  mkdir build && cd build
- cmake -DGT_ENABLE_BACKEND_MC=ON -DGT_ENABLE_BACKEND_CUDA=ON -DCMAKE_INSTALL_PREFIX=/usr/local ..
- make install -j4
- make test
+ cmake -DCMAKE_INSTALL_PREFIX=/usr/local ..
+ cmake --build --parallel 4
+ ctest
 
-|GT| uses CMake as building system. The installation can be configured using `ccmake`. The following variables control the back-ends that will be supported by the runtime components of the installation, namely :ref:`GCL <halo-exchanges>`.
+CMake will detect the optional dependencies and enable the available backends accordingly.
+During configure the available GridTools targets will be listed.
 
-.. code-block:: shell
+The following CMake variables are available to customize the installation of |GT|.
 
- GT_ENABLE_BACKEND_CUDA # For CUDA GPUs
- GT_ENABLE_BACKEND_X86  # For cache based multicores
- GT_ENABLE_BACKEND_NAIVE  # For naive implementation
- GT_ENABLE_BACKEND_MC   # For optimized multicores and KNL
+  * Set ``GT_INSTALL_EXAMPLES`` to ``ON`` and
+  * select a directory for installation with ``GT_INSTALL_EXAMPLES_PATH``.
+    The examples come with a standalone CMake project and can be built separately.
 
-All the targets can be installed and used at the same time, but some runtime components may lead to incompatibilities or complex environments to make the codes run. It may be more effective to do multiple installs of the library for different targets in this case.
+Additionally, use the following CMake variables to customize building of tests.
 
-At the moment the only runtime component of |GT| is the GCL, the communication module. By default, this component will be installed in the system as a single-node mockery of the full distributed memory capability. To enable the full GCL you must set to ``ON`` the following variable
+  * Set ``BUILD_TESTING`` to ``OFF`` to disable building any tests (fast installation without validation).
+  * Set ``GT_GCL_GPU`` to ``OFF`` to disable the ``gcl_gpu`` target (and to disable building of GPU GCL tests).
+    This is useful, if you have CUDA in your environment, but the MPI implementation is not CUDA-aware.
+  * Set ``GT_CUDA_ARCH`` to the compute capability of the GPU device on which you want to run the tests.
+  * If your compiler is a CUDA-capable *Clang*, you can switch how CUDA code will be compiled, by setting
+    ``GT_CLANG_CUDA_MODE`` to one of ``AUTO`` (default, prefer Clang-CUDA if available), ``Clang-CUDA`` (compile with
+    Clang), ``NVCC-CUDA`` (compile with NVCC and Clang as the host compiler).
 
-.. code-block:: shell
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Using a GridTools installation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
- GT_USE_MPI
+Using |GT| follows standard CMake practices. If |GT| was installed to `<prefix>`,
+provide ``-DCMAKE_PREFIX_PATH=<prefix>`` or ``-DGridTools_ROOT=<prefix>`` to indicate where |GT| can be found.
```

Comment:
Is this still the path to the `cmake` subfolder?

Reply:
No